unity-environment/Assets/ML-Agents/Scripts/Agent.cs

```diff
@@ -756,6 +756,20 @@ public void UpdateTextAction(string textActions)
     {
         action.textActions = textActions;
     }
+    
+    /// <summary>
+    /// Scales continous action from [-1, 1] to arbitrary range.
+    /// </summary>
+    /// <param name="rawAction"></param>
+    /// <param name="min"></param>
+    /// <param name="max"></param>
+    /// <returns></returns>
+    protected float ScaleAction(float rawAction, float min, float max)
+    {
+        var middle = (min + max) / 2;
+        var range = max - min;
```

Comment:
I think this is `var range = (max - min) / 2`
Also, I don't think this is needed, or it should go into a utils class ?